crates/topos-tce/src/lib.rs

```diff
@@ -47,11 +47,7 @@ pub async fn run(
     };
 
     let message_signer = match &config.signing_key {
-        Some(AuthKey::PrivateKey(pk)) => {
-            let bytes = pk.to_vec();
-            let bytes_str = std::str::from_utf8(&bytes)?;
-            MessageSigner::new(bytes_str)?
-        }
+        Some(AuthKey::PrivateKey(pk)) => MessageSigner::new(&hex::encode(pk))?,
```

Comment:
Was this the UTF8 related error we saw at start up?

Reply:
Yes seems so